### ISSUE TITLE
Missing version bound on turtle breaks build

### ISSUE THREAD

**Issue by glguy** [apauley/hledger-flow#109]
Turtle had a breaking change in 1.6. Until hledger-flow is updated to work with the new filepath types, an upper bound will be needed.

**Comment by apauley** [comment]
Hi @glguy - could you give the full context of what you are doing and what is failing?

To build this project I'm running the build command listed in the [build instructions](https://github.com/apauley/hledger-flow/blob/master/CONTRIBUTING.org#build-the-project). 

Are you using the same commands, or something else? Please provide the commands together with any error output. Additional context such as your operating system etc may also be helpful.

**Comment by gleber** [comment]
Yes, that is correct. As soon as I updated resolver to 21.13, a lot of build failures due to Turtle file path handling change have started. I've made it to work with Turtle 1.6.1 and all remaining changes in stackage lts-21.13 at #114

**Comment by apauley** [comment]
Fixed as part of #114
